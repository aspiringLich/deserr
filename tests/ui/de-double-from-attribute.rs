use deserr::Deserr;

#[derive(Deserr)]
struct UnitStruct {
    #[deserr(from(String) = String::parse -> usize)]
    #[deserr(from(String) = usize::FromStr -> usize)]
    hello: usize,
}

fn main() {}
