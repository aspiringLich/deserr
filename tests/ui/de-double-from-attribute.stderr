error: The `try_from` field attribute is defined twice.
 --> tests/ui/de-double-from-attribute.rs:6:14
  |
6 |     #[deserr(try_from(String) = usize::FromStr -> usize)]
  |              ^^^^^^^^
